README.md

```diff
@@ -137,6 +137,8 @@ The executable can be found in the build/release/bin folder.
   `brew install cmake`
 
   `brew install zeromq`
+  
+  `brew install hidapi`
```

Comment:
This doesn't seem to exist upstream. Please send a pull request there and get it approved.